eval/run_generated_agent_evaluation.py

```diff
@@ -13,22 +13,34 @@
 
 
 async def run_evaluation(
-    evaluation_case_json_file: str = "generated_workflows/latest/evaluation_case.json",
-    agent_trace_json_file: str = "generated_workflows/latest/agent_eval_trace.json",
-    save_evaluation_results_path: str = "generated_workflows/latest/evaluation_results.json",
+    generated_workflow_dir: str = "generated_workflows/latest",
+    evaluation_case_json_file: str | None = None,
+    agent_trace_json_file: str | None = None,
+    save_evaluation_results_path: str | None = None,
```

Comment:
kept these mostly for backwards compatibility, but we can probably just get rid of them